### PR TITLE
fix discography in artist page

### PR DESCRIPTION
**Pull request by esibruti**
**Details of Changes**
Add details of changes here.
- Fixed Expand button.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by itsWindows11** [comment]
Do you have a screenshot?

**Comment by esibruti** [comment]
@itsWindows11
https://user-images.githubusercontent.com/78214358/189532180-c5481093-1260-4f73-83c7-aeec02c4ac7a.mp4

**Comment by itsWindows11** [comment]
nice

File: Rise Media Player Dev/Views/Artists/ArtistSongsPage.xaml [hunk 1]
```diff
@@ -319,6 +311,15 @@
                                     </MenuFlyout>
                                 </AppBarButton.Flyout>
                             </AppBarButton>
+
+                            <AppBarButton
+                                x:Name="UpDown"
+                                Click="UpDown_Click"
+                                Label="Collapse">
+                                <AppBarButton.Icon>
+                                    <FontIcon x:Name="UpDownIcon" Glyph="&#xE010;" />
```
Comment (itsWindows11) [inline Rise Media Player Dev/Views/Artists/ArtistSongsPage.xaml:320]: No need for the x:Name here since we aren't using it in code behind

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
I have requested some changes above

**Review by itsWindows11 — APPROVED** [review]

**Pull request merged**